### DISCUSSION TITLE
plot size

### DISCUSSION THREAD

**Discussion by Jiajia-TaoTao** [kcleal/gw#67] [Q&A] [answered]
Can I resize the image? I didn't see it in the explanation

**Comment by kcleal** [comment, marked as answer]
Yes, there are a few ways:
1. Use the mouse to change the window size
2. Set the dimensions of the image using `--dims` on command line or `General -> dimensions` in
 the settings menu (also click the save button to make changes permanent).

![image](https://github.com/user-attachments/assets/888f4727-1879-4c12-b616-f66d8aa2ea3b)

3. Use `SHIFT + ARROW-KEY` short cut
4. Write the dimension size in the .gw.ini file usually located under ~/.config/.gw.ini
